axon.go

```diff
@@ -125,7 +125,8 @@ func (a *Axon) Run() error {
 	}
 
 	// Create a notify listener and start from the configured changeset id.
-	listener := NewNotifyListener(StartFromID(a.Config.StartFromID))
+	listenerLog := a.Logger.WithFields(logrus.Fields{"component": "listener"})
```

Comment:
Adding the component field could be done "by default" inside the listener.

Reply:
Ah true, at the start.

Reply:
done